kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -416,8 +416,10 @@ def _report_container_spec_metrics(self, pod_list, instance_tags):
                 if self.pod_list_utils.is_excluded(cid, pod_uid):
                     continue
 
-                tags = instance_tags[:]
-                tags += tagger.tag('%s' % replace_container_rt_prefix(cid), tagger.HIGH) or []
+                tags = tagger.tag('%s' % replace_container_rt_prefix(cid), tagger.HIGH) or []
```

Comment:
I think we can just drop the `or []` since it was used to avoid appending to a `None` object.
same for the rest.

Reply:
Good call, it breaks a lot of checks now.

Reply:
Good point, actually None was specified explicitely here: https://github.com/DataDog/integrations-core/blob/master/kubelet/datadog_checks/kubelet/kubelet.py#L369-L371
I'll follow the same pattern